WordPressAuthenticatorTests/OAuth/ProofKeyForCodeExchangeTests.swift

```diff
@@ -0,0 +1,24 @@
+@testable import WordPressAuthenticator
+import XCTest
+
+class ProofKeyForCodeExchangeTests: XCTestCase {
+
+    func testCodeChallengeInPlainModeIsTheSameAsCodeVerifier() {
+        XCTAssertEqual(
+            ProofKeyForCodeExchange(codeVerifier: "abc", mode: .plain).codeCallenge,
+            "abc"
+        )
+    }
+
+    func testCodeChallengeInS256ModeIsEncodedAsPerSpec() {
+        // TODO:
+    }
```

Comment:
I put this down as a double reminder to finish the PKCE implementation. See also [previous comment](https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/717/files#r1043267507).